libraries/AP_Avoidance/AP_Avoidance.h

```diff
@@ -96,6 +92,8 @@ class AP_Avoidance {
     static const struct AP_Param::GroupInfo var_info[];
 
 protected:
+    // constructor
```

Comment:
miss the public create function no? otherwise we won't be able to create the object, right?